tests/legacy-cli/e2e/tests/build/ssr/express-engine-ngmodule.ts

```diff
@@ -103,16 +103,9 @@ export default async function () {
           const style = await browser.driver.findElement(by.css('style[ng-app-id="ng"]'));
           expect(await style.getText()).not.toBeNull();
 
-          // Test the contents from the server.
```

Comment:
Can you please change the expect please? As removing this, removes an important part of the tests,  that validates that the SSR view is rendered without an error